src/Components/Publishing/Partner/PartnerInline.tsx

```diff
@@ -77,10 +77,10 @@ const PartnerInlineContainer = styled.div`
   }
   ${pMedia.sm`
     img {
-      max-height: 24px;
+      max-height: 32px;
     }
     ${Icon} {
-      font-size: 24px;
```

Comment:
Why are we removing the mobile font-size for the icon?

Reply:
I looked at this in mobile and saw the same case (a smaller sponsor logo in mobile). I figured they would also want the change reflect there as well.

Reply:
But wouldn't we want to make the sponsor logo match the 24 height, rather than keeping them both at the desktop size?

Reply:
The sponsor logo was 24px but the Artsy logo was 32px. I changed the height of the sponsor logo to match the Artsy logo which I thought was the ask. Are you suggesting to change the Artsy logo to be 24px in mobile?

Reply:
Isn't `${Icon}` the artsy logo?  it looks like this line removes a smaller size for the artsy logo in mobile.

Reply:
🤦‍♀️  you're totally right. I definitely overlooked that.